src/signals/incident-management/containers/IncidentDetail/components/StatusForm/styled.tsx

```diff
@@ -2,85 +2,114 @@
 // Copyright (C) 2020 - 2021 Gemeente Amsterdam
 import styled from 'styled-components'
 import {
+  Alert,
   Heading,
-  themeSpacing,
+  Label,
   Row,
-  Column,
   themeColor,
-  Label,
+  themeSpacing,
 } from '@amsterdam/asc-ui'
 
 import Button from 'components/Button'
 
+export const AddNoteWrapper = styled.div`
+  label {
+    display: none;
+  }
+  section div textarea {
+    padding: ${themeSpacing(3)};
+    margin-top: 0;
+    border-top: 1px solid transparent;
+    :hover {
+      border-top-color: transparent;
+    }
+  }
+`
+
 export const Form = styled.form`
   position: relative;
+  padding: ${themeSpacing(0, 5, 6, 5)};
+  margin-bottom: ${themeSpacing(6)};
+  background-color: ${themeColor('tint', 'level2')};
   width: 100%;
   grid-template-areas:
     'header'
     'options'
     'texts'
     'form';
-  grid-column-gap: 20px;
   display: grid;
-  margin-bottom: 5em;
   grid-row-gap: ${themeSpacing(4)};
+`
 
-  @media (min-width: ${({ theme }) => theme.layouts.medium.max}px) {
-    grid-template-columns: 6fr 6fr;
-    grid-template-areas:
-      'header texts'
-      'options texts'
-      'form texts'
-      '. texts';
-  }
+export const FormArea = styled.div`
+  grid-area: form;
+  display: grid;
+  grid-row-gap: ${themeSpacing(6)};
 `
 
 export const HeaderArea = styled.div`
   grid-area: header;
 `
 
-export const StyledLabel = styled(Label)`
-  font-weight: 700;
-`
-
 export const OptionsArea = styled.div`
   grid-area: options;
   display: grid;
   grid-row-gap: ${themeSpacing(4)};
 `
 
-export const TextsArea = styled.div`
-  grid-area: texts;
-  margin-top: ${themeSpacing(5)};
+export const QuestionLabel = styled.div`
+  margin: ${themeSpacing(2)} 0;
 `
 
-export const FormArea = styled.div`
-  grid-area: form;
-  display: grid;
-  grid-row-gap: ${themeSpacing(6)};
+export const StandardTextsButton = styled(Button)`
+  margin-top: ${themeSpacing(2)};
+  border-bottom: none;
+  border-color: ${themeColor('tint', 'level5')};
+  padding: ${themeSpacing(3, 3, 0, 3)};
+  width: 100%;
+  :hover {
+    outline-style: none;
+  }
+  div {
+    font-weight: normal;
+    font-family: 'Avenir Next';
+    text-align: left;
+    width: 100%;
+    height: 100%;
+    padding-bottom: ${themeSpacing(3)};
+    border-bottom: 1px solid ${themeColor('tint', 'level4')};
+  }
 `
 
-export const QuestionLabel = styled.div`
-  margin: ${themeSpacing(2)} 0;
+export const StyledAlert = styled(Alert)<{ level?: string }>`
```

Comment:
`level` Can't be just a string when it already expects a value of `neutral`. Can it be that level is of type [`AlertLevel`](https://github.com/Amsterdam/amsterdam-styled-components/blob/2ec8deae5aca1bff0796824979a461c4e488471d/packages/asc-ui/src/components/Alert/AlertStyle.ts)?